### PR TITLE
Add support for WFS and ArcGIS Rest

### PR DESCRIPTION
**Pull request by bitner**

### CODE REVIEW

File: osgeo_importer/importers.py [hunk 1]
```diff
@@ -223,14 +225,16 @@ def import_file(self, *args, **kwargs):
 
         # Add index for any layers configured by name
         for layer_configuration in configuration_options:
-            lookup = 'layername'
-
-            if lookup not in layer_configuration and 'index' in layer_configuration:
+            if 'layer_name' in layer_configuration:
```
Comment (garnertb) [inline osgeo_importer/importers.py]: `lookup` needs to be declared with a default value or the for loop below will throw an exception if both `if` statements are passed.

File: osgeo_importer/inspectors.py [hunk 2]
```diff
@@ -168,28 +168,36 @@ def describe_fields(self):
...
-            layer_definition = layer.GetLayerDefn()
-
-            for i in range(layer_definition.GetFieldCount()):
-                field_desc = {}
-                field = layer_definition.GetFieldDefn(i)
-                field_desc['name'] = field.GetName()
-                field_desc['type'] = field.GetFieldTypeName(i)
-                layer_description['fields'].append(field_desc)
+            else:
+                layer_description = {'layer_name': layer_name,
```
Comment (garnertb) [inline osgeo_importer/inspectors.py]: Is there a way to make this more DRY?  This method should return a standard dictionary regardless of the driver.  If certain values don't make sense for the driver then they should be `None`.

File: osgeo_importer/inspectors.py [hunk 3]
```diff
@@ -174,21 +174,25 @@ def describe_fields(self):
         for n in range(0, opened_file.GetLayerCount()):
             layer = opened_file.GetLayer(n)
             layer_name = layer.GetName()
+            try:
+                geometry_type = self.geometry_type(layer.GetGeomType())
```
Comment (garnertb) [inline osgeo_importer/inspectors.py]: What is the exception that gets thrown here?  The `geometry_type` should handle any errors and return `None` by default.

**Comment by bitner** [comment]
Items fixed as per comments from  @garnertb

**Comment by jj0hns0n** [comment]
@garnertb can we get a review and merge?

**Comment by garnertb** [comment]
Rebased and merged in 6a9b34.

**Pull request closed without merging**